### ISSUE TITLE
Docs test doesn't run in T2 created cluster

### ISSUE THREAD

**Issue by fhennig** [stackabletech/nifi-operator#442]
It runs locally as well as on a manually created GCP cluster, but not in the T2 created GCP cluster. @sbernauer and I suspect that it might be due to firewall config.

**Comment by lfrancke** [comment]
This is probably not relevant anymore I assume @sbernauer

**Comment by sbernauer** [comment]
Yeah, I think this was related to network security stuff. We don't use T2 any more, all fine
